server/lib/get-contract-by-id.js

```diff
@@ -107,7 +107,8 @@ module.exports = exports = async (contractId, locals = {}) => {
 		contract = reformatSalesforceContract(contract);
 		contract.last_updated = new Date();
 		if (contract.orders) {
-			const activeOrder = contract.orders.find(order => order.status === 'Active');
+			const now = new Date();
```

Comment:
**Nitpick (non-blocking)**: `currentTimeInMilliseconds` is another cool name to refer to `now`